### ISSUE TITLE
dl: removal of `downloader` dir caused network traffic on monitoring

### ISSUE THREAD

**Issue by AskAlexSharov** [erigontech/erigon#10976]
i have node which have all files. 
after delete `downloader` dir - in logs:
```
[INFO] [07-02|10:30:25.606] [1/6 Snapshots] downloading header-chain progress="12.00% 192.2GB/1.6TB" time-left=3hrs:14m total-time=23m40s download=123.5MB/s upload=102.4KB/s peers=1107 files=524 metadata=524/524 connections=2779 alloc=10.0GB sys=15.5GB
```

in monitoring (produced by node, not by erigon): 
<img width="1570" alt="Screenshot 2024-07-02 at 17 31 57" src="https://github.com/ledgerwatch/erigon/assets/46885206/1f240583-fb79-49c9-902e-56c4579a4407">

**Comment by AskAlexSharov** [comment]
after 3 days (7 restarts) nodes not finished downloading:  
```
ethmainnet: 
[INFO] [07-05|04:45:57.711] [1/6 Snapshots] downloading header-chain progress="70.99% 1.1TB/1.6TB" time-left=1321hrs:51m total-time=9m40s download=102.4KB/s upload=102.4KB/s peers=682 files=524 metadata=524/524 connections=1180 alloc=8.4GB sys=12.3GB

sepolia: 
[INFO] [07-05|04:47:07.408] [1/6 Snapshots] downloading header-chain progress="22.87% 115.4GB/504.4GB" time-left=29010291hrs:30m total-time=9m0s download=4B/s upload=0B/s peers=79 files=178 metadata=178/178 connections=903 alloc=2.4GB sys=2.9GB
```

**Comment by AskAlexSharov** [comment]
I fixed download stuck by: https://github.com/ledgerwatch/erigon/pull/11037

**Comment by awskii** [comment]
closed by #11037
